spec/models/refinery/jobs/job_spec.rb

```diff
@@ -16,6 +22,40 @@ module Jobs
           expect(FactoryGirl.build(:job, {description: 1, title: 1})).to be_valid
         end
       end
+
+      context 'update' do
+        it 'should be functional' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.